erpnext/maintenance/doctype/maintenance_schedule/maintenance_schedule.py

```diff
@@ -237,26 +239,30 @@ def create_maintenance_opportunity(maintenance_schedule, row):
 	default_opportunity_type = frappe.get_cached_value("CRM Settings", None, "default_opportunity_type_for_schedule")
 	schedule = schedule_doc.getone('schedules', {'name': row})
 
-	opportunity_doc = frappe.new_doc('Opportunity')
+	if not schedule:
+		frappe.throw(_("Row does not exist in Maintenace Schedule"))
```

Comment:
Spelling mistake. Lets improve the error message to
"Invalid Maintenance Schedule Row Provided"